Src/Sarif.PatternMatcher/Validation.cs

```diff
@@ -51,7 +51,7 @@ public enum Validation
         /// <summary>
         /// The host is unknown.
         /// </summary>
-        HostUnknown,
+        UnknownHost,
```

Comment:
>UnknownHost [](start = 8, length = 11)

Oops, changed the plug in enum and didn't adjust this one. I'll add a unit test for this next.